app/src/main/res/values/dimens.xml

```diff
@@ -378,5 +378,6 @@
     <dimen name="lux_chart_height">200dp</dimen>
 
     <dimen name="lux_chart_axis_text">9sp</dimen>
+    <dimen name="multimeter_quantity_dimen">45sp</dimen>
```

Comment:
You have ticked that there are no additions at the end of resource files to avoid conflicts and here is the only addition at the end of `dimens.xml`